x-pack/plugins/osquery/public/live_queries/form/pack_queries_status_table.tsx

```diff
@@ -737,15 +786,33 @@ const PackQueriesStatusTableComponent: React.FC<PackQueriesStatusTableProps> = (
     }
   }, [agentIds?.length, data, getHandleErrorsToggle, itemIdToExpandedRowMap]);
 
+  const queryIds = useMemo(
+    () =>
+      map(data, (query) => ({
+        value: query.action_id || '',
+        field: 'action_id',
+      })),
+    [data]
+  );
+  const casesOwner = useMemo(() => [SECURITY_SOLUTION_OWNER, OBSERVABILITY_OWNER], []);
```

Comment:
Looks like there are some references to attaching to timeline, observability does not support timeline. Do we need to support attaching osquery results to observability?

Reply:
Not yet, I am not sure if there is a timeline for that. Indeed, there is one place in application to still take care of, thanks for pointing this out 👍

Reply:
Got rid of that bug, thank you :)

Reply:
@patrykkopycinski Is o11y aware of the integration?